index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Page</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+            <h1 class="header">John Doe</h1>
+    
+            <nav>
+                <img src="assets/images/menuIcon.png" alt="" class="icon menu-icon">
+                <div class="nav-links" style="display: none;">
+                    <a href="" id="portfolio">Portfolio</a>
+                    <a href="" id="about">About</a>
+                    <a href="" id="contact">Contact</a>
+                </div>
+            </nav>
+    </header>
+    <main>
+        <section class="hero">
+            <span class="header">Hey There.
+                I’m Alex</span>
+            <span class="sub-header">I’am a Software Developer</span>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="social-icons">
+                <a href="https://github.com/Reggeditt" class="social-link">
+                    <img src="assets/images/githubIcon.png" alt="" class="icon">
+                </a>
+                <a href="https://www.linkedin.com/in/tetteh-kodjo-sarso-888989125/" class="social-link">
+                    <img src="assets/images/linkedin.png" alt="" class="icon">
+                </a>
+                
+                <a href="https://twitter.com/tksarso" class="social-link">
+                    <img src="assets/images/twitterIcon.png" alt="" class="icon">
+                </a>
+            </div>
```

Comment:
- [ ] Good job adding the social links 👍🏽 . As the previous reviewer suggested, you should add all the social media icons in the Figma design 😄. I understand you don't have some of them now, it is okay. You can add them as placeholders and route them with # for now pending the time you register on the other platforms 👍🏽 .

e.g  

 ```
 <a href="#" class="social-link">
         <img src="assets/images/angellistcon.png" alt="" class="icon">
    </a>
```

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/204862993-cfa9fdbe-dd10-4fad-9dd2-2d84e9845a60.png)

Figma design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/204863234-2545754d-9e81-4777-b41f-70aa88d374c7.png)

Reply:
fixed @vickymarz